Portal/Helpers/TagHelpers/RadioTagHelper.cs

```diff
@@ -42,17 +42,17 @@ public override async Task ProcessAsync(TagHelperContext context, TagHelperOutpu
 
             var dataProperty = AspFor;
 
-            if (AspDataFor != null)
-                dataProperty = AspDataFor;
+//            if (AspDataFor != null)
```

Comment:
Do we want to keep these commented out lines?

Reply:
Good spot. Done